src/form/Select/Select.tsx

```diff
@@ -197,6 +198,11 @@ export interface SelectProps {
    * Menu component override.
    */
   menu?: ReactElement<SelectMenuProps, typeof SelectMenu>;
+
+  /**
+   * The options for the Fuse.js search algorithm.
+   */
+  fuseOptions?: Fuse.IFuseOptions<SelectOptionProps>;
```

Comment:
Let's decouple this name from the impelmentation. Make it more like `searchOptions` or something.